<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>391</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* style.qss */
QGroupBox {
    background-color: white;
    border: 2px solid #0C455B; /* Blue border */
    border-radius: 5px;
}
QPushButton {
    background-color: white;
    border:1px solid #0C455B;
    border-radius: 10px;
    color: black;
	font: 6.5pt &quot;Poppins&quot;;
}

QPushButton:hover {
    background-color: #e0e0e0; /* Change to a lighter shade when hovered */
}

QPushButton:pressed {
    background-color: #bfbfbf; /* Change to a darker shade when pressed */
}
#mdp {
background-color:transparent;
    border: none;
    font: 81 7pt &quot;Poppins&quot;;
    text-decoration: underline;
    
}

QPushButton:hover {
    color: #032f81; /* Change color on hover */
}

/*#piclogin {
    background-image: url(&quot;E:/Esprit/Esprit 2.0/Project c++ (login)/Login/icon.png&quot;);
    background-position: center;
    background-repeat: no-repeat;
}

#picmdp {
    background-image: url(&quot;E:/Esprit/Esprit 2.0/Project c++ (login)/Login/key.png&quot;);
    background-position: center;
    background-repeat: no-repeat;
}*/

QLineEdit {
    background-color: transparent;
    border: 1px solid #0C455B; /* Add a border to QLineEdit */
    border-radius: 3px;
    padding: 3px; /* Add padding for better appearance */
    
}

QLineEdit:hover {
    border-color: #0C455B; /* Change border color on hover */
}
</string>
     </property>
     <property name="title">
      <string>Sign In</string>
     </property>
     <widget class="QLineEdit" name="username_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mdp_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe oubliéé</string>
      </property>
     </widget>
     <widget class="QLabel" name="piclogin_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>74</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/icon/icon/compte.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="picmdp_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>114</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/icon/icon/pass.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm_3">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>391</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="windowTitle">
      <string>MainWindow</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: white;
    border: 2px solid #0C455B; /* Blue border */
    border-radius: 5px;
}

QPushButton {
    border: 2px solid #0C455B; /* Blue border */
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #e0e0e0; /* Change to a lighter shade when hovered */
}</string>
     </property>
     <property name="title">
      <string>Sign In</string>
     </property>
     <widget class="QLabel" name="Picture_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>51</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/icon/icon/logologin.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>183</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
